cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -30,11 +30,11 @@ Application::
 An application is a program designed for end users. An application consists of multiple microservices or components that work individually to build the entire application.
 Examples of applications: a video game, a media player, a web browser.
 Component::
-A component is a building block of an application. Typically it is a software package, a web service, a web resource, or a module that encapsulates a set of related functions or data. 
+A component is a set of kubernetes resources which host code or data. Each component can be run and deployed separately.
```

Comment:
@dharmit @kadel @Preeticp - Have made the suggested changes. Please review. Thanks

Reply:
Looks good. Thanks @nkakkar81 !

Reply:
LGTM 
@nkakkar81 please remove the WIP, in the description right at the top, mention that the PR has been reviewed by SME and peer reviewed by the DevTools team. Post that, ping Andrea either here or in the team docs channel and ask for a review and merge, please.

Reply:
@bergerhoffer - This PR has been reviewed by the SME, and the peer review is also completed by the DevTools Team. Request you to please review and merge ( if there are no comments). Thanks!

Reply:
kubernetes -> Kubernetes